cognite/client/_api/simulators/simulation_runs.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from cognite.client._api_client import APIClient
+from cognite.client._constants import DEFAULT_LIMIT_READ
+from cognite.client.data_classes.simulators.filters import SimulationRunsFilter
+from cognite.client.data_classes.simulators.simulators import SimulationRun, SimulationRunsList
+from cognite.client.utils._experimental import FeaturePreviewWarning
+
+if TYPE_CHECKING:
+    from cognite.client import ClientConfig, CogniteClient
+
+
+class SimulatorRunsAPI(APIClient):
+    _RESOURCE_PATH = "/simulators/runs"
+
+    def __init__(self, config: ClientConfig, api_version: str | None, cognite_client: CogniteClient) -> None:
+        super().__init__(config, api_version, cognite_client)
+        self._warning = FeaturePreviewWarning(
+            api_maturity="General Availability", sdk_maturity="alpha", feature_name="Simulators"
+        )
+
+    def list(
+        self, limit: int = DEFAULT_LIMIT_READ, filter: SimulationRunsFilter | dict[str, Any] | None = None
+    ) -> SimulationRunsList:
+        """`Filter simulation runs <https://developer.cognite.com/api#tag/Simulation-Runs/operation/filter_simulation_runs_simulators_runs_list_post>`_
+
+        List simulation runs
+
+        Args:
+            limit (int): The maximum number of simulation runs to return.
```

Comment:
```suggestion
            limit (int): Maximum number of results to return. Defaults to 1000. Set to -1, float(“inf”) or None to return all items.
```

Reply:
I suspect this will auto paginate if limit is set to `None`, `-1` or `float("inf")`, following the normal Python SDK pattern